kubejs/server_scripts/base/tags/items/forge/stone.js

```diff
@@ -0,0 +1,5 @@
+onEvent('item.tags', (event) => {
+    stones.forEach((stone) => {
+        event.add("forge:stone", stone);
```

Comment:
This event accepts arrays, so you can just say `event.add("forge:stone", stones)`